<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>540</width>
    <height>364</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>-30</x>
      <y>0</y>
      <width>621</width>
      <height>381</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/cct/Mosaic.jpg);</string>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_info_4">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>20</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 12pt &quot;Garamond&quot;;
background-color: rgb(29, 116, 255);
color: rgb(54, 54, 80);</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:18pt; font-weight:600; color:#ffffff;&quot;&gt;AES-RSA CIPHER&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_info">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>110</y>
      <width>241</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 2pt &quot;Garamond&quot;;
background-color: rgb(29, 116, 255);
color: rgb(54, 54, 80);</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:600; color:#ffffff;&quot;&gt;Select a file for encrypt or decrypt:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="txt_file">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>150</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Garamond</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Garamond&quot;;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_loadfile">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>150</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Garamond</family>
      <pointsize>11</pointsize>
      <weight>9</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 11pt &quot;Garamond&quot;;
background-color: rgb(29, 116, 255);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Load Text File...</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_info_2">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>110</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 12pt &quot;Garamond&quot;;
background-color: rgb(29, 116, 255);
color: rgb(54, 54, 80);</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600; color:#ffffff;&quot;&gt;Function:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cmb_function">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>150</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 12pt &quot;Garamond&quot;;</string>
    </property>
    <property name="currentText">
     <string>Select Function</string>
    </property>
    <property name="placeholderText">
     <string/>
    </property>
    <item>
     <property name="text">
      <string>Select Function</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Encrypt</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Decrypt</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="btn_start">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>290</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Garamond</family>
      <pointsize>14</pointsize>
      <weight>9</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 14pt &quot;Garamond&quot;;
background-color: rgb(29, 116, 255);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Start!</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_info_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>210</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 12pt &quot;Garamond&quot;;
background-color: rgb(29, 116, 255);
color: rgb(54, 54, 80);</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600; color:#ffffff;&quot;&gt;Digital Signature Service:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cmb_signature">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>240</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 12pt &quot;Garamond&quot;;</string>
    </property>
    <property name="currentText">
     <string>Select Operation</string>
    </property>
    <property name="placeholderText">
     <string/>
    </property>
    <item>
     <property name="text">
      <string>Select Operation</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Sign</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Verify</string>
     </property>
    </item>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="mosaic.qrc"/>
 </resources>
 <connections/>
</ui>
